rails/Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+
+gem 'benchmark-ips', github: 'kirs/benchmark-ips', branch: 'quiet-mode'
+gem 'rails', path: '/rails'
+gem 'ffaker'
+gem 'sqlite3'
+gem 'stackprof'
+gem 'mysql2'
+gem 'pg'
+gem 'arel', github: 'rails/arel', branch: 'master'
```

Comment:
We don't need to fix, because Rails master is also built against HEAD gems: https://github.com/rails/rails/blob/master/.travis.yml#L6